extensions/amp-analytics/0.1/variables.js

```diff
@@ -236,6 +248,10 @@ export class VariableService {
     this.register_('SCROLL_LEFT', () =>
       Math.round(Services.viewportForDoc(this.ampdoc_).getScrollLeft())
     );
+
+    this.register_('EXPERIMENT_BRANCHES', () =>
+      experimentBranchesMacro(this.ampdoc_.win)
```

Comment:
Does the experiment branches differ in FIE or shadow doc? What's the current status on ampdoc-fie, does `this.ampdoc_.win` refer to the FIE window or the parent window?

Reply:
I think ampdoc fie is near launching. For shadowDoc it looks like win will point to the parent window. For FIE for now this will be also the parent window, but will change when ampdoc fie launched to be the iframe win, at which point the parent experiments will not be accessible. Im not sure what the right behavior is here, do you think we should always return the parent experiments in FIE case? If so we can do something like `this.ampdoc_.getParent().win  || this.ampdoc_.win`

Reply:
Reviewed the current usage, looks like an ad specific feature. 
I don't have a preference, but looks like currently all usage are in the parent window. Two questions
1. Is there a plan to scope experiment to within the FIE? For example, some css change that apply to a single FIE. 
2. Is it OK to pass the parent docs experiment data to FIE? Especially if the experiment is unrelated to the FIE.

Reply:
yeah these are good questions. @lannka & @powerivq for thoughts.

Reply:
Talked offline, we decided to always look at the parent window. Changes made.

Reply:
> Is it OK to pass the parent docs experiment data to FIE? Especially if the experiment is unrelated to the FIE.

Just want to double confirm on this.

Reply:
yep better to be safe, @lannka @powerivq the way this is now written is that FIE can access the parent win experiments through this macro now and after FIE doc is launched. Can we confirm this is the behavior we want?

Reply:
that should be fine. there shouldn't be any secret in what experiment the code is on. it's purely random